mmdet/core/bbox/assigners/atss_assigner.py

```diff
@@ -70,6 +84,23 @@ def assign(self,
 
         # compute iou between all bbox and gt
         overlaps = self.iou_calculator(bboxes, gt_bboxes)
+        
+        # compute cost rate for proposal only in DDOD
+        if self.alpha != None:
```

Comment:
```python
if self.alpha == None:
     # ATSSAssigner
     overlaps = self.iou_calculator(bboxes, gt_bboxes)
else:
     # Dynamic cost ATSSAssigner in DDOD
      assert cls_scores is not None and bbox_preds is not None, 'Invalid ``alpha`` parameter because ``cls_scores`` or ``bbox_preds`` are None. If you want to use the cost-based ATSSAssigner,  please set ``cls_scores``, ``bbox_preds`` and ``self.alpha`` at the same time. '
      ...